mobile/src/config/navigation-theme.config.ts

```diff
@@ -0,0 +1,17 @@
+import { Theme } from '@react-navigation/native';
+
+import { AppColor } from '~/common/enums/enums';
+
+const NavigationTheme: Theme = {
```

Comment:
```suggestion
const navigationTheme: Theme = {
```

Reply:
I wanted to name it with a small letter, then I spied on a mini project, and there with a capital letter